src/components/Tab/src/Tab.stories.tsx

```diff
@@ -23,11 +23,11 @@ const Template: Story<TabProps> = (args: TabProps) => {
     <TabContext value={value.toString()}>
       <Tabs
         value={value}
-        onChange={(event: React.ChangeEvent<unknown>, newValue: number) => {
+        onChange={(_event: React.ChangeEvent<unknown>, newValue: number) => {
```

Comment:
Why was this renamed?

Reply:
To fix a typecheck error:
![err](https://user-images.githubusercontent.com/22854131/120892735-06c0b100-c610-11eb-9d6b-a11fac854a5a.PNG)
Eslint provided a quickfix suggesting this solution, to signal to other developers that the parameter is unused in the function.